### DISCUSSION TITLE
getting this message

### DISCUSSION THREAD

**Discussion by shfraser777** [osy86/HaC-Mini#606] [Q&A] [answered]
“HaCMini.pkg” cannot be opened because it is from an unidentified developer.
 macOS cannot verify that this app is free from malware.

Safari downloaded this file today at 1:57 PM from github.com.

I am running Bug Sur 11.4
New to Mac, New To Hackintosh

**Comment by shfraser777** [comment]
<img width="614" alt="Screen Shot 2021-07-21 at 3 11 57 PM" src="https://user-images.githubusercontent.com/87673839/126546240-fa9352d2-1045-4655-ac21-3ffe3156d772.png">
got past the problem above.  now have another problem
 it skipped over destination type and installed on my local drive rather than my USB.  Two questions.

1. how to I delete it from my local drive
2. how do I get the hacmini not to skip the destination select option?

**Comment by osy** [comment, marked as answer]
https://osy.gitbook.io/hac-mini-guide/installation-guide/installation

**Comment by shfraser777** [comment]
Thanks. I’ve been following this but some of the instructions seem dated.  

The command to create the installation volume has an error in that it fails to indicate that installer should be replaced by the name out your usb. 

There is also another problem with the HacMini package, as it skips over the step where you chose your destination.  And there is no option to chose on the following step.     

Probably the guide was written for Catalina and the app was updated but the instructions were not. 

Thus I ended up installing it on my laptop’s internal drive.  Now trying to find out how to reverse that before doing a reboot.

**Reply by osy** [reply]
>  it fails to indicate that installer should be replaced by the name out your usb.

From the linked guide:

> Give the name Installer and make sure to select GUID Partition Map as the Scheme.

> as it skips over the step where you chose your destination.  And there is no option to chose on the following step

From the linked guide:

> Press Continue and then Customize

Please read more carefully :P

**Comment by shfraser777** [comment]
I got through Creating OS X installer OK and now have a USB Drive with Big Sur installation file on it.

But Patching the OS X Installer is where I have run into problems.


<img width="619" alt="Screen Shot 2021-07-21 at 5 28 04 PM" src="https://user-images.githubusercontent.com/87673839/126563626-d0a94b3a-87f5-4619-ab8e-7b9db462e4ac.png">
<img width="623" alt="Screen Shot 2021-07-21 at 5 29 07 PM" src="https://user-images.githubusercontent.com/87673839/126563804-5cf2c6eb-0b1b-4ff7-852b-62c02e577902.png">
<img width="601" alt="Screen Shot 2021-07-21 at 5 29 31 PM" src="https://user-images.githubusercontent.com/87673839/126563818-2bf60326-4074-4b5f-b665-e333dec0dc89.png">
<img width="517" alt="Screen Shot 2021-07-21 at 5 30 19 PM" src="https://user-images.githubusercontent.com/87673839/126563838-1e69bca1-eea2-4bf6-a6f2-2ed2f7739e18.png">
<img width="624" alt="Screen Shot 2021-07-21 at 5 30 48 PM" src="https://user-images.githubusercontent.com/87673839/126564190-ec27bcb8-cd95-4323-bc3b-ed3d6075169a.png">
<img width="605" alt="Screen Shot 2021-07-21 at 5 31 25 PM" src="https://user-images.githubusercontent.com/87673839/126564217-91a69b13-c2e1-4a1d-927e-f05db3247063.png">

it skips the destination select and customize shows nothing about changing install location ?

What am I missing?  But most importantly, how do I back out the insulation from my local hard drive?
 Thanks in advance.

**Reply by shfraser777** [reply]
Arggghh. Chance install location!! It was right there!!  facepalm.

Ok. Last question.  How do I back this erroneous installation off of my hard drive before I reboot?
